src/main/webapp/app/controllers/schemaManagementController.js

```diff
@@ -42,6 +44,12 @@ cap.controller("SchemaManagementController", function($controller, $scope, Schem
   $scope.cancelCreateSchema = function() {
     angular.extend($scope.schemaToCreate, SchemaRepo.getScaffold());
     $scope.resetSchemaForms();
+    $scope.resetCreateSchemaPropertyList = false;
+    var timer = $timeout(function(){
```

Comment:
Because there is a directive inside the modal. 

This isn't like the other forms being reset. It isn't even part of a form.

The form in that modal is being reset in the traditional way.

Reply:
I am going to try it without the timeout, to avoid race conditions.

Reply:
I did try that. Maybe it will work for you.